README.md

```diff
@@ -110,13 +114,15 @@ This repository contains two dataset folders with examples of data to run the co
   - sample-dataset: Contains just one pair of features/labels with some default values. This data lets you run the code in demo mode.
   - protein-folding: Contains pairs of features/labels for various proteins (prepared using https://github.com/simonholmes001/structure_prediction). The features represent protein characteristics, and the labels the distance between all aminoacids.
 
-### 9. Environment variables
+### 9. Grid search and inference parameters
       
-The model and grid search can be set up using a set of environment variables contained in the grid-search-parameters.sh. Please refer to the ENVIRONMENT_VARIABLES.md for the full list of available environment variables and how to use them.
+The grid search and inference parameters can be set up using a parameter json file contained in the parameters directory. Please refer to the PARAMETERS.md for the full list of available parameters and how to use them.
 
 ### 10. Execute a grid search
        
-Before executing a grid-search please go to the grid-search.sh to add your PYTHONPATH=path/to/message-passing-nn/.
+Before executing a grid-search please go to the grid-search.sh to add your:
```

Comment:
I feel the need for a web application for set-up........

Reply:
Get your ass to work! (After we merge what you already have)